money/currency.py

```diff
@@ -401,7 +401,7 @@ class CurrencyHelper:
             'display_name': 'Colombian Peso',
             'numeric_code': 170,
             'default_fraction_digits': 2,
-            'sub_unit': 100,
```

Comment:
Facebook might be wrong on this one. The link @shridharama provided and Wikipedia both indicate that 100 is correct.

Reply:
It's possible Facebook isn't "wrong" here per se but is just setting arbitrary rules for the minimum allowed bid on their ad units, which sometimes matches up with established convention for that currency and sometimes doesn't (see [here](https://developers.facebook.com/docs/marketing-api/currencies/#example-offset-1) in the docs). In that case, if the sub unit isn't defunct we may want to go with the conventional usage as opposed to Facebook's rules. Thoughts?

Reply:
Ooh good find, @kaylafuchs. @matthewsnell, maybe we should leave Colombia and Costa Rica as they were and take your changes on the others?